core/src/main/java/com/datastax/oss/driver/api/core/DriverInfo.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core;
+
+/** Exposes information about the driver. */
+public interface DriverInfo {
```

Comment:
"Info" is a bit generic. What this really is is the artifact coordinates.

Reply:
As you pointed out below, we might want to enrich this interface with more info than just the GAV coordinates, which is why I chose the word "Info".